packages/link-element/LinkElement.js

```diff
@@ -4,14 +4,15 @@ export default function LinkElement({ html, state }) {
     return html`
 <style>
   :host {
-    color: var(--primary-500);
+    color: var(--accent);
```

Comment:
aXe and Lighthouse report contrast issues with this color in light and dark mode.